buildSrc/src/main/groovy/com/uber/okbuck/rule/android/AndroidTestRule.groovy

```diff
@@ -40,6 +40,7 @@ final class AndroidTestRule extends AndroidRule {
                 aptDeps,
                 providedDeps,
                 aidlRuleNames,
+                null,
```

Comment:
I marked the PR as a work in progress for that reason. Wanted to get this out early for early review and so others could begin to use it.